### DISCUSSION TITLE
How can I install a previous version of Vercel (19.0.1)

### DISCUSSION THREAD

**Discussion by edcamposs** [vercel/vercel#7638] [Help] [answered]
Hi, I'm following an Udemy course where they are installing Vercel 19.0.1 and I need to install this version because in last version I can't init "custom-build" project.

Thanks in advance!!

**Comment by leerob** [comment, marked as answer]
Hey, could you share more about what you're trying to build? You could probably `npm i -g vercel@19.0.1`, but I'm unsure what "custom-build" means in this context and if it will work.

**Reply by edcamposs** [reply]
I'm not building anything yet. I'm just learning. I found the files at Github. Thanks for answer.
